draft-ietf-lake-edhoc-psk.md

```diff
@@ -149,7 +153,7 @@ When using an external PSK, a common representation of CRED_I and CRED_R is a CB
 }
 ~~~~~~~~~~~~
 
-Alternative formats for CRED_I and CRED_R MAY be used. When a resumption PSK is employed, CRED_I and CRED_R MUST be the same credentials used in the initial EDHOC exchange, for example, public-key credentials such as X.509 certificates.
+Alternative formats for CRED_I and CRED_R MAY be used. When a resumption PSK is employed, CRED_I and CRED_R MUST refer to the same long-term credentials as in the initial EDHOC exchange, for example, public-key credentials such as X.509 certificates. The identifier associated with the resumption PSK (e.g., a kid) can change between resumptions, but it continues to resolve to the same underlying credential.
```

Comment:
"CRED_I and CRED_R MUST refer to the same long-term credentials as in the initial EDHOC exchange"
I have problems with this change. We want to allow different formats for CRED_x, it may contain the PSK or not. The terminology is still that CRED_x are the "authentication credenitals" and ID_CRED_PSK is referring to the authentcation credentials. Saying that CRED_x refers to credentials therefore becomes confusing. I think the original formulation is correct: We discussed different options for maintaining disambiguation and agreed on the simple method to always use the initial CRED_I and CRED_R, and this applies even in resumption when PSK and kid are changed.